core/src/main/scala/com/nvidia/spark/rapids/tool/planparser/WindowGroupLimitParser.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2024, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids.tool.planparser
+
+import com.nvidia.spark.rapids.tool.qualification.PluginTypeChecker
+
+import org.apache.spark.sql.execution.ui.SparkPlanGraphNode
+
+case class WindowGroupLimitParser(
+    node: SparkPlanGraphNode,
+    checker: PluginTypeChecker,
+    sqlID: Long) extends ExecParser {
+
+  val fullExecName: String = node.name + "Exec"
+
+  override def parse: ExecInfo = {
+    // WindowGroupLimit doesn't have duration
+    val duration = None
+    val exprString = node.desc.replaceFirst("WindowGroupLimit ", "")
+    val expression = SQLPlanParser.parseWindowGroupLimitExpression(exprString)
+    val notSupportedExprs = checker.getNotSupportedExprs(expression.toSeq)
```

Comment:
somehwere here the code should check for the list of accepted expressions.
This is a common case that we may need to do for other Execs. For example, @nartal1 and I were just discussing the same thing for #715 
So, it will be good if the implementation we come up with here can easily be used to fix #715

Reply:
Added an `validateRankingExpr()` for this current use case. The refactoring could be taken up in a separate PR.